src/simulator/AccelerometerModule.tsx

```diff
@@ -113,10 +113,13 @@ const Gesture = ({ icon, state, enabled, onValueChange }: GestureProps) => {
         value={choice}
         onChange={handleSelectChange}
         ref={selectRef}
+        fontFamily="code"
       >
         {choices.map((choice) => (
           <option key={choice} value={choice}>
-            {choice}
+            {intl.formatMessage({
+              id: `simulator-gesture-${choice.split(" ").join("-")}`,
+            })}
```

Comment:
I guess you could make this work for the final state now by showing it bracketed if it's different from the English? I think that's what we intend, though not sure which way round. Please can you follow up on that discussion and check there's consensus?

Reply:
Unable to render any HTML in the option tag, so the font family will be the same -> [`4fc19e8`](https://github.com/microbit-foundation/python-editor-v3/pull/993/commits/4fc19e859dee39451d9a7fee4c674ee2c4ab5e27).